e2e/config/config.go

```diff
@@ -71,6 +91,22 @@ type Options struct {
 	Deployers []string
 }
 
+// Default namespace mappings for Kubernetes (k8s) clusters.
+var k8sNamespaces = Namespaces{
+	RamenHubNamespace:  "ramen-system",
+	DRClusterNamespace: "ramen-system",
```

Comment:
RamenDRClusterNamespace

A bit long, but consistent with the RamenHubNamespace